lib/ansible/modules/database/postgresql/postgresql_facts.py

```diff
@@ -0,0 +1,916 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+
+DOCUMENTATION = '''
+---
+module: postgresql_facts
+short_description: Gather facts about remote PostgreSQL servers
+description:
+   - Gathers facts about remote PostgreSQL servers.
+version_added: "2.8"
+options:
+  incl_subset:
+    description:
+      - Limit collected subsets by comma separated string.
+      - Allowable values are C(version),
+        C(databases), C(settings), C(tablespaces), C(languages), C(roles), C(namespaces),
+        C(replications), C(repl_slots), C(extensions).
+      - By default, collects all subsets.
+      - You can use shell-style (fnmatch) wildcard to pass groups of values (see Examples).
+    default: "*"
+    type: list
+  excl_subset:
+    description:
+      - Exclude subsets from the collection.
+      - Allowable values are similar as in I(incl_subset).
+      - You can also use shell-style (fnmatch) wildcard.
+      - Mutually exclusive with incl_subset.
+    type: list
+  db:
+    description:
+      - Name of database to connect.
+  port:
+    description:
+      - Database port to connect.
```

Comment:
```suggestion
      - Database port to connect.
    type: int
```
Please do this for all the parameters.